runtime/doc/deprecated.txt

```diff
@@ -36,6 +36,11 @@ DIAGNOSTICS
 - The "cursor_position" parameter of |vim.diagnostic.JumpOpts| is renamed to
   "pos"
 
+TREESITTER
+• *TSNode:child_containing_descendant()*	Use
+  |TSNode:child_with_descendant()| instead; it is identical except that it can
+  return the descendant itself.
```

Comment:
🚀  "with" is also less verbose, we should never use "contains" (use "has" instead) or "containing" (use "with" instead) in a function name.

Reply:
This is directly from upstream.

Reply:
Good job @amaanq :)